packages/postgres-database/src/simcore_postgres_database/models/services.py

```diff
@@ -70,6 +78,16 @@
     sa.Column(
         "write_access", sa.Boolean, nullable=False, server_default=expression.false()
     ),
+    sa.Column(
+        "product_name",
+        sa.String,
+        sa.ForeignKey(
+            "products.name",
+            name="fk_services_name_products",
+            onupdate="CASCADE",
+            ondelete="CASCADE",
```

Comment:
does this mean that if we delete a product, the service is deleted, right?
Then, a service X to be deployed in say 3 products will have 3 entries in this table.
This gives us the advantage of even defining different access rights based on the product
Do I understand it right?

Reply:
yes that is correct and is exactly the use of these modifications